app/assets/javascripts/discourse/tests/integration/components/plugin-outlet-test.js

```diff
@@ -0,0 +1,151 @@
+import { module, test } from "qunit";
+
+import { count, exists, query } from "discourse/tests/helpers/qunit-helpers";
+import { setupRenderingTest } from "discourse/tests/helpers/component-test";
+
+import { click, render, settled } from "@ember/test-helpers";
```

Comment:
```suggestion
import { module, test } from "qunit";
import { count, exists, query } from "discourse/tests/helpers/qunit-helpers";
import { setupRenderingTest } from "discourse/tests/helpers/component-test";
import { click, render, settled } from "@ember/test-helpers";
```

(there's a PR/issue already in eslint or prettier for removing extra whitespace between imports 😄)